pdp/dao.py

```diff
@@ -66,27 +94,35 @@ def get_name(netid):
     return name
 
 
-def get_student(netid):  # TODO fix this to use student id
+# uwhr is default--we can override to use HEPPS for now
+def get_employee(eid, source='uwhr'):
     """
-    Look up IRWS student data by netid (for mock...system key in real life).
+    Look up IRWS employee data by eid
+    defaults to uwhr--override to hepps until HRP finishes
     """
     try:
-        # TODO fix this to use student id
-        sdb_person = IRWS().get_sdb_person(sid=netid)
-        if not sdb_person:
-            raise ServiceError('no SDB entry')
+        # person = IRWS().get_person() # don't need?
+        uwhr_person = IRWS().get_uwhr_person(eid=eid)
+        if source == 'hepps':
+            uwhr_person = IRWS().get_hepps_person(eid=eid)
```

Comment:
No.  Do we need this to be able to work with hepps?  technically uwhr isn't for production until HRP goes live.